### PR TITLE
Added tool for operating single file, added documentation, added markdown

### PR DESCRIPTION
**Pull request by judovana**
Hello!

I recently ended up with preparing a notes for a slide show and realized that the presentation is actually done... So I used the WeakPoint to run it. However some tweeks were necessary:
  *  the work over multiple files was clumsy - added support for singl efile
  *  some highlight is necessary - enabled markdown
  * documented it all

Looking forward for review!

### CODE REVIEW

File: WeekPointSplitter.lua [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+local function boolToString(b)
+    if (b) then
+        return "true"
+    else
+        return "false"
+    end
+end
+local function help(exit)
+    print("Spliting uttility for github.com/tisnik/vim-weakpoint (ideally with github.com/plasticboy/vim-markdown [which neds .markdown suffix])")
```
Comment (tisnik) [inline WeekPointSplitter.lua:27]: some typos :)

File: WeekPointSplitter.lua [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+local inFile = nil
+local outDir = OUT_DIR_DEFAULT
+local clean = false
+local deduct = false
+local height = 0
+local page = "--PAGE--"
+local vim = false
+
+local function boolToString(b)
+    if (b) then
```
Comment (tisnik) [inline WeekPointSplitter.lua:20]: iirc the parenthesis are not needed there

File: WeekPointSplitter.lua [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    local r = currentPage..""
+    while (#r < width) do
+        r="0"..r;
+    end
+    r=r..suffix;
+    io.stderr:write(r.."\n");
+    return r;
+end
+
+if (#arg == 0) then
```
Comment (tisnik) [inline WeekPointSplitter.lua:109]: dtto, not needed parenthesis

**Review by tisnik — APPROVED** [review]

**Comment by tisnik** [comment]
Hi @judovana. It looks very very good. Thanks a lot! There are just minor issues, but I don't want to bother you with them at this moment.

**Comment by judovana** [comment]
Hello!

Thank you for check. I confess, I'm not going to fix the brackets:( I have issues with code without them:(

As for
Spliting->Splitting
uttility->utility
neds->needs

I will fix asap.

**Pull request merged**